common/upgets/api_resource_spot_finder.lua

```diff
@@ -0,0 +1,317 @@
+local upget = gadget or widget
+local globalScope = gadget and _G or WG
```

Comment:
perhaps GG instead of _G (since that's the equivalent of WG)? alternatively could just be _G since it's present in both

Reply:
I see both GG and _G in the codebase, seems like some unification effort for the global should be done. Probably happened due to pulling shit from different codebases without putting much thought into it.